python/ray/train/_internal/backend_executor.py

```diff
@@ -111,6 +111,10 @@ def start(
             actor_cls_kwargs=train_cls_kwargs,
             placement_group=placement_group,
         )
+        logger.info(
+            f"Starting {self._trial_info.name} on data parallel workers: "
+            f"{[w.metadata.node_ip for w in self.worker_group.workers]}."
+        )
```

Comment:
Looking through the [design](https://gist.github.com/scottsun94/b98e3a54adba1b60b381ea958069fd33) I have a few comments:

1. It would be nice to include the process ID in addition to node_ip, but not sure what the best way is to format it without making it look like a port ID (cc @scottsun94).
2. I don't think the Trial name is useful for Train.
3. Technically these do not have to be data parallel, so I would avoid adding this detail.

Reply:
for training - 

changed to `(TorchTrainer pid=80906) 2023-04-19 08:58:37,959	INFO backend_executor.py:117 -- Starting distributed workers: ['127.0.0.1:80923', '127.0.0.1:80924', '127.0.0.1:80925']`


for tuning - 
`(TorchTrainer pid=81746) 2023-04-19 09:01:03,976	INFO backend_executor.py:121 -- Starting TorchTrainer_5f8b5_00000 on distributed workers: ['127.0.0.1:81780', '127.0.0.1:81781', '127.0.0.1:81782'].`

Reply:
127.0.0.1:80923 this looks really like ip+port...

Here are some answers from gpt 4:

> The best format for showing an IP address and process ID together is to display them in a clear and easily readable manner. You can use a combination of delimiters, such as colons, hyphens, or brackets to separate the two pieces of information. Here are a few examples:

IP_Address:Process_ID (e.g., 192.168.1.100:12345)
IP_Address[Process_ID] (e.g., 192.168.1.100[12345])
IP_Address-Process_ID (e.g., 192.168.1.100-12345)
It's important to choose a format that is both clear and consistent throughout your application or documentation. You can also include a brief explanation or legend for users if the format is not immediately obvious.

Reply:
How about this?
`Starting distributed worker processes: ['80923 (127.0.0.1)', '80924 (127.0.0.1:)', '80925 (127.0.0.1:)']`

Reply:
Updated.

Reply:
Very nice!